server/src/main/resources/reference-docker.conf

```diff
@@ -9,6 +9,11 @@ modeldb: {
   metadataDb: {
     "host" : "MONGODB_HOST",
     "port" : 27017,
+
+    # If you want a username and password for mongo:
+    # "username": "my_username",
+    # "password": "${PASSWORD}",
```

Comment:
This should be `MONGODB_PASSWORD`, and you should add a comment noting that it'll be read from an environment variable.

You should also add these lines to the other configuration files.

Reply:
Made this fix and added it to other config files